app/assets/javascripts/controllers/auth_key_pair_cloud/auth_key_pair_cloud_controller.js

```diff
@@ -1,69 +0,0 @@
-ManageIQ.angular.app.controller('keyPairCloudFormController', ['$http', '$scope', 'keyPairFormId', 'miqService', function($http, $scope, keyPairFormId, miqService) {
-  var vm = this;
-
-  var init = function() {
-    vm.keyPairModel = {
-      name: '',
-      public_key: '',
-      ems_id: '',
-    };
-
-    vm.formId = keyPairFormId;
-    vm.afterGet = false;
-    vm.modelCopy = angular.copy( vm.keyPairModel );
-    vm.model = 'keyPairModel';
-    vm.ems_choices = [];
-    vm.saveable = miqService.saveable;
-    vm.newRecord = keyPairFormId === 'new';
-
-    ManageIQ.angular.scope = $scope;
-
-    miqService.sparkleOn();
-    $http.get('/auth_key_pair_cloud/ems_form_choices')
```

Comment:
The `ems_form_choices` from the `AuthKeyPairCloudController` can be also deleted, together with its route.